### DISCUSSION TITLE
Completely unreasonable request:

### DISCUSSION THREAD

**Discussion by mousethatischarles** [Alex313031/Thorium-Android#43] [Ideas]
I have a couple of devices I'm rather attached to stuck with old versions of Android:
JXDS7800B - Android 4.2.2 (a still surprisingly nice Android emulation tablet)
Blackberry Passport - BBOS 10 Android 4.2.2 compatibility (by far my favourite phone, aging compatibility is becoming a real issue)

Unreasonable request: I've had no luck getting Thorium to install on either of these, I presume it's understandably compiled to a later version of the OS. It would be awesome if a release was done for older Android.*

*The still pretty vibrant BBOS community would probably love you forever - the modern-ish browser options are getting really limited.

Oh, and Extension support for the Android versions would be awesome.

Many thanks.

PS: Running Thorium on all my systems that can - by far the best browser I've used.

**Comment by gz83** [comment]
I am afraid that it is no longer of great value for new versions of browsers to support Android 4 series systems, and many technologies are no longer supported on Android 4 series systems.
